.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "liboqs"]
```

Comment:
AFAIK, we don't "like" submodules -- or would you be OK with this here, @dstebila @SWilson4 @vsoftco ?

Reply:
I'm fine with removing the submodules. This will require some documentation updates to reflect the changes. Let me know if you'd prefer this approach.

Reply:
I'd rather not have the submodule if we can help it. The language wrapper should "just work" with new versions of liboqs as long as the core APIs don't change.

Reply:
Agreed; our approach so far has been to not include liboqs as a submodule.